cobc/cobc.c

```diff
@@ -90,6 +90,24 @@ enum compile_level {
 	CB_LEVEL_EXECUTABLE	= 7
 };
 
+#define	CB_FLAG_GETOPT_STACK_SIZE       1
```

Comment:
Maybe those should be in flag.def, directly before the matching CB_FLAG or at its header?

Reply:
`flag.def` is included several times into `cobc.c`, so I was not confortable with these macros being included several times, maybe generating warnings on some systems.

Reply:
That multi-inclusion is totally right, so cobc.c is the right place.